webapp/channels/src/packages/mattermost-redux/src/actions/search.ts

```diff
@@ -19,7 +19,7 @@ import {receivedFiles} from './files';
 import {forceLogoutIfNecessary} from './helpers';
 import {getMentionsAndStatusesForPosts, receivedPosts} from './posts';
 
-const WEBAPP_SEARCH_PER_PAGE = 20;
+export const WEBAPP_SEARCH_PER_PAGE = 20;
```

Comment:
I don't see this getting used anywhere else. Why the export?

Reply:
I exported it for use in some tests. `searchFilesWithParams` didn't have tests of its own before, but it was covered by tests for `searchFiles` which wasn't used outside of those tests, so I got rid of `searchFiles` and updated its tests to call `searchFilesWithParams` directly